lib/auth/permissions.go

```diff
@@ -210,6 +211,16 @@ func GetCheckerForBuiltinRole(role teleport.Role) (services.AccessChecker, error
 					services.KindAuthServer: services.RO(),
 				},
 			})
+	case teleport.RoleTrustedCluster:
+		return services.FromSpec(
+			role.String(),
+			services.RoleSpecV2{
+				Namespaces: []string{services.Wildcard},
+				Resources: map[string][]string{
+					services.KindAuthServer:    services.RO(),
+					services.KindCertAuthority: services.RW(),
```

Comment:
it means that it can write all cert authorities, not just it 's CA that is a vulnerabillity